setup.cfg

```
@@ -31,7 +31,7 @@ install_requires =
     types-protobuf>=3.17.3
     typing-extensions
     packaging
-    treevalue>=1.4
+    optree>=0.7.0
```

```suggestion
    optree>=0.6.0
```

version 0.6.0 should be enough.